### PR TITLE
Bp cted 2178

### PR DESCRIPTION
**Pull request by BrendonPretorius**

### CODE REVIEW

File: lib.php [hunk 1]
```diff
@@ -8484,12 +8484,10 @@ function hsuforum_view($forum, $course, $cm, $context) {
  */
 function hsuforum_discussion_view($modcontext, $forum, $discussion) {
     
-    if (isset($_SESSION['mod_hsuforum_page_refresh']) && $_SESSION['mod_hsuforum_page_refresh'] > 0) {
+    $pageWasRefreshed = isset($_SERVER['HTTP_CACHE_CONTROL']) && $_SERVER['HTTP_CACHE_CONTROL'] === 'max-age=0';
```
Comment (JJSwanevelder) [inline lib.php:8487]: This will not work with safari

Comment (BrendonPretorius) [reply lib.php:8487]: It's the reverted stuff that Rumano did in the hsu_forum submodule/

Comment (JJSwanevelder) [reply lib.php:8487]: Gotcha

**Review by JJSwanevelder — APPROVED** [review]
👍

**Pull request merged**